middleware/src/middleware.go

```diff
@@ -748,7 +748,24 @@ func (middleware *Middleware) EnableRootLogin(toggleAction rpcmessages.ToggleSet
 	return rpcmessages.ErrorResponse{Success: true}
 }
 
-// SetRootPassword sets the systems root password
+// EnablePasswordLogin enables/disables the ssh login with a password (in addition to ssh keys)
+// and returns a ErrorResponse indicating if the call was successful.
+func (middleware *Middleware) EnablePasswordLogin(toggleAction rpcmessages.ToggleSettingArgs) rpcmessages.ErrorResponse {
```

Comment:
Adding `EnablePasswordLogin` here is not enough. See https://github.com/shiftdevices/bitbox-base-internal/issues/269 for an overview.